tests/scalers_go/openstack_swift/helper.go

```diff
@@ -0,0 +1,58 @@
+//go:build e2e
```

Comment:
I'd name this file openstack_helper, but let's dont block on this.

Reply:
I had problems with naming in packages, if I name this `openstack_helper`, the linters push me to name the other `openstack_helper_test` and I prefer to no modify the linter rules for this

Reply:
😄 leave it as it it then